common/dev_disconnect.c

```diff
@@ -29,7 +29,9 @@ static void next_dev_disconnect(void)
 		 dev_disconnect_line, sizeof(dev_disconnect_line)-1);
 	if (r < 0)
 		err(1, "Reading dev_disconnect file");
-	lseek(dev_disconnect_fd, -r, SEEK_CUR);
+	if (lseek(dev_disconnect_fd, -r, SEEK_CUR) == -1) {
```

Comment:
I usually prefer to compare `< 0` rather than `== -1` for POSIX interfaces.  See line 30 above for the return value comparison of `read` for example, despite the fact that `read` returns exactly `-1` on error.

Reply:
@ZmnSCPxj Updated! Now using `< 0` for consistency :-)